build_luajit.sh

```diff
@@ -26,6 +26,8 @@ fi
 ljbins=${ljbins:=builds}
 
 mkdir -p ${ljbins}
+# Make directory path relative so we don't have to worry about what path we have to pass to make.
+ljbins="$( cd "$ljbins" && pwd )"
```

Comment:
`${ljbins}` for consistnecy?

What does `$(...)` do here? Never seen this before.

Reply:
I assume capture the stdout of the command run inside it.

Reply:
Ah subshell. Fair enough.

I always use backticks, but this is fine too.